management/src/clusterm/manager/commission_event.go

```diff
@@ -82,16 +84,27 @@ func (e *commissionEvent) process() error {
 	return nil
 }
 
+func (e *commissionEvent) eventValidate() error {
+	var err error
+	if !IsValidHostGroup(e.hostGroup) {
+		return errored.Errorf("host-group is a mandatory parameter and is not specified %s", e.hostGroup)
```

Comment:
I think this error will also come when specified host-group is an invalid string.

May be we shall reword the error as:

`invalid or empty host-group specified: %q`